doc/yaml-reference.md

```diff
@@ -165,8 +165,65 @@ image: gcc:5.1
 image: !from_yaml .gitlab-ci.yml image
 ```
 
+Here's a more elaborate example which defines multiple aliases which correspond
+to jobs defined by `.gitlab-ci.yml`:
 
-[YAML]: http://yaml.org/
-[`.gitlab-ci.yml`]: http://doc.gitlab.com/ce/ci/yaml/README.html
+**`.gitlab-ci.yml`**
+```yaml
+
+build_c:
+  image: gcc:5.1
+  script:
+    - make something
+    - make something-else
+
+build_py:
+  image: python:3.7
+  script:
+    - setup.py bdist_wheel
+```
+
+**`.scuba.yml`**
+```yaml
+image: not-specified (see issue #124)
+
+aliases
+  build_c:
+    image: !from_yaml .gitlab-ci.yml build_c.image
+    script: !from_yaml .gitlab-ci.yml build_c.script
+  build_py:
```

Comment:
I think it would be helpful here to show that (at least for Gitlab CI configuration files), one can directly import the entire job block:
```yaml
aliases:
  build_py: !from_yaml .gitlab-ci.yml build_py
```
For experienced users of `scuba` this pattern is obvious, but since this doc is targeted to newer users, it might be helpful to have an example of this pattern. I would either change this alias in the example, or add a third.